README.md

```diff
@@ -30,7 +30,7 @@ We are left with;
 
 $T(n) \leq g(\log_{5} n)$, $T(n) \leq h(\log_{2} n)$
 
-Now using the rule that $O$ ignores constants we have;
+Because the definition of $O$ says we can ignore constants;
```

Comment:
How does the definition say that?

Reply:
Oh, I thought that you were saying that I was using the wrong terminology in that last comment you left, or are you looking for something along the lines of; big O ignores constants because even though they can be important it only looks at the overall growth, so in our case a log function is still a log no matter what constant is being multiplied

Reply:
Formally the argument is that the $c$ allows us to have any constant, which is why the constant doesn't matter. This is what we used in class, and you'd have to use something similar for the proof. In particular, you can't just say that the definition allows us to ignore constants.

Reply:
Alright, that makes sense, thank you, I'll get my proof fixed.